src/httpserver.cpp

```diff
@@ -665,3 +669,14 @@ void UnregisterHTTPHandler(const std::string &prefix, bool exactMatch)
     }
 }
 
+std::string urlDecode(const std::string &urlEncoded) {
+    std::string res;
+    if (!urlEncoded.empty()) {
```

Comment:
`evhttp_uridecode` works with an empty string, as such this `if` can be removed.